Can't make deploy

**Describe the bug**
A clear and concise description of what the bug is.

/home/k8s-master/istio-operator
git branch
origin/release-1.4

```
root@k8s-master:/home/k8s-master/istio-operator# kubectl get nodes
NAME         STATUS   ROLES    AGE   VERSION
k8s-master   Ready    master   8d    v1.17.0
k8s-node01   Ready    <none>   8d    v1.17.0
k8s-node02   Ready    <none>   8d    v1.17.0
```

when i  ``` make deploy ```

```
root@k8s-master:/home/k8s-master/istio-operator# make deploy
scripts/install_kustomize.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   640  100   640    0     0    639      0  0:00:01  0:00:01 --:--:--   640
100 18.8M  100 18.8M    0     0   229k      0  0:01:24  0:01:24 --:--:--  108k
mv: cannot stat 'kustomize_*_linux_amd64': No such file or directory
chmod: cannot access 'bin/kustomize': No such file or directory
Makefile:63: recipe for target 'bin/kustomize' failed
make: *** [bin/kustomize] Error 1
```

```
root@k8s-master:/home/k8s-master/istio-operator# ls
ADOPTERS.md  cmd                 config           deploy      Dockerfile.dev  Gopkg.lock  hack                         LICENSE   pkg      README.md
bin          CODE_OF_CONDUCT.md  CONTRIBUTING.md  Dockerfile  docs            Gopkg.toml  kustomize_2.0.3_linux_amd64  Makefile  PROJECT  scripts
```

```
root@k8s-master:/home/k8s-master/istio-operator# ll
total 19428
drwxr-xr-x 14 root       root           4096 May 21 03:24 ./
drwxr-xr-x 28 k8s-master k8s-master     4096 May 21 01:44 ../
-rw-r--r--  1 root       root            390 May 21 01:45 ADOPTERS.md
drwxr-xr-x  2 root       root           4096 May 21 03:11 bin/
drwxr-xr-x  2 root       root           4096 May 21 01:46 .circleci/
drwxr-xr-x  3 root       root           4096 May 21 01:45 cmd/
-rw-r--r--  1 root       root           3217 May 21 01:45 CODE_OF_CONDUCT.md
drwxr-xr-x  5 root       root           4096 May 21 01:46 config/
-rw-r--r--  1 root       root            664 May 21 01:45 CONTRIBUTING.md
drwxr-xr-x  3 root       root           4096 May 21 01:45 deploy/
-rw-r--r--  1 root       root            778 May 21 01:45 Dockerfile
-rw-r--r--  1 root       root            783 May 21 01:45 Dockerfile.dev
-rw-r--r--  1 root       root              4 May 21 01:45 .dockerignore
drwxr-xr-x  4 root       root           4096 May 21 01:46 docs/
-rw-r--r--  1 root       root            212 May 21 01:45 .editorconfig
drwxr-xr-x  8 root       root           4096 May 21 01:47 .git/
drwxr-xr-x  3 root       root           4096 May 21 01:45 .github/
-rw-r--r--  1 root       root            451 May 21 01:45 .gitignore
-rw-r--r--  1 root       root            332 May 21 01:45 .golangci.yml
-rw-r--r--  1 root       root          31205 May 21 01:45 Gopkg.lock
-rw-r--r--  1 root       root           2478 May 21 01:45 Gopkg.toml
drwxr-xr-x  2 root       root           4096 May 21 01:45 hack/
drwxr-xr-x  2 root       root           4096 May 21 01:45 .idea/
-rw-r--r--  1 root       root       19723712 May 21 03:25 kustomize_2.0.3_linux_amd64
-rw-r--r--  1 root       root          11356 May 21 01:45 LICENSE
-rw-r--r--  1 root       root            279 May 21 01:45 .licensei.toml
-rw-r--r--  1 root       root           4596 May 21 01:45 Makefile
drwxr-xr-x 12 root       root           4096 May 21 01:45 pkg/
-rw-r--r--  1 root       root             80 May 21 01:45 PROJECT
-rw-r--r--  1 root       root           8147 May 21 01:46 README.md
drwxr-xr-x  2 root       root           4096 May 21 01:46 scripts/
```

```
root@k8s-master:/home/k8s-master/istio-operator# uname -r
5.3.0-42-generic
root@k8s-master:/home/k8s-master/istio-operator# uname -a
Linux k8s-master 5.3.0-42-generic #34~18.04.1-Ubuntu SMP Fri Feb 28 13:42:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
root@k8s-master:/home/k8s-master/istio-operator# 
```

**Steps to reproduce the issue:**
Please describe the steps to reproduce the issue.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem like release numberm version, branch, etc.

--- a/scripts/install_kustomize.sh
+++ b/scripts/install_kustomize.sh
@@ -8,5 +8,5 @@ curl -O -L "https://github.com/kubernetes-sigs/kustomize/releases/download/v${ve
 
 # move to bin
 mkdir -p bin
-mv "kustomize_*_${opsys}_amd64" bin/kustomize
+mv "kustomize_${version}_${opsys}_amd64" bin/kustomize
 chmod u+x bin/kustomize

